support.md

```diff
@@ -24,9 +24,9 @@ There are also a few extra medium-specific guidelines, listed with each type of
 ### Reporting Violations
 
 Should you need to speak with someone about another Foreman member's behaviour,
-email the Community Lead, *Greg Sutcliffe* (greg.sutcliffe at gmail.com). You
-may also CC or email *Ohad Levy* (ohad.levy at gmail.com).  We'll do our best
-to listen to your issue fairly and take appropriate action.
+email the Community Lead, *Greg Sutcliffe* (greg at emeraldreverie.org, male)
```

Comment:
It may be a bit odd to have a title for you but not for Ori.

Reply:
True, I'll delete mine, it's not relevant anyway

Reply:
Done in the Github UI as im on another branch right now, so its an extra commit. squash on merge pls :P